staticaddr/script/script.go

```diff
@@ -0,0 +1,113 @@
+package script
```

Comment:
nit: ideally we have some basic tests for these scripts. There are examples of how to do that in `swap/htlc_test.go`.

Reply:
Done, wrote test cases in `script_test.go` for success/error of coop and timeout spend.